test/services/contract_wrappers/sheltering_wrapper.js

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright: Ambrosus Technologies GmbH
+Email: tech@ambrosus.com
+
+This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+This Source Code Form is “Incompatible With Secondary Licenses”, as defined by the Mozilla Public License, v. 2.0.
+*/
+
+import chai from 'chai';
+import sinon from 'sinon';
+import sinonChai from 'sinon-chai';
+import chaiAsPromised from 'chai-as-promised';
+import ShelteringWrapper from '../../../src/services/contract_wrappers/sheltering_wrapper';
+
+chai.use(sinonChai);
+chai.use(chaiAsPromised);
+const {expect} = chai;
+
+describe('Sheltering Wrapper', () => {
+  let contractManagerMock;
+  let shelteringWrapper;
+
+  describe('isSheltering', () => {
+    let isShelteringStub;
+    let isShelteringCallStub;
+    const bundleId = 'bundle';
+    const defaultAddress = '0x123';
+
+    beforeEach(async () => {
+      isShelteringStub = sinon.stub();
+      isShelteringCallStub = sinon.stub();
+      contractManagerMock = {
+        shelteringContract: async () => ({
+          methods: {
+            isSheltering: isShelteringStub
+          }
+        }),
+        defaultAddress: () => defaultAddress
+      };
+      isShelteringStub.returns({
```

Comment:
You can shorten it like in `configWrapper` tests. Actually every contractWrapper class should have this shortened like in `configWrapper` tests.